index.html

```diff
@@ -4596,78 +4596,80 @@ <h2>Key and Signature Expiration</h2>
+Performing revocation immediately after rotation is useful for verification
+methods that a controller designates for short-lived verifications, such as
+those involved in encrypting messages and authentication.
       </p>
 
       <p>
-        A controller performs a rotation when they add a new verification
-        method that is meant to replace an existing verification method after
-        some time.
+The following considerations might be of use when contemplating the use of
+<a>verification method</a> rotation:
       </p>
 
-      <p>Not all DID Methods support verification method rotation.</p>
-
-      <p>
-        Rotation is a key management process that enables the private cryptographic
-        material associated with an existing verification method to be deactivated
-        or destroyed once a new verification method has been added to the DID
-        Document. Going forward, any new proofs that a controller would have generated
-        using the old cryptographic material can now instead be generated using the new
-        material and can be verified using the new verification method.
-      </p>
-
-      <p>
-        Rotation is a useful mechanism for protecting against verification
-        method compromise, since frequent rotation of a verification method by
-        the controller reduces the value of a single compromised verification method
-        to an attacker. Performing revocation immediately after rotation is
-        useful for verification methods that a controller designates for
-        short-lived verifications, such as those involved in encrypting messages
-        and authentication.
-      </p>
-
-      <p class="note">
-        Higher security environments tend to employ more frequent verification
-        method rotation.
-      </p>
-
-      <p class="note">
-        Frequent rotation of a verification method might be frustrating for
-        parties that are forced to continuously renew or refresh associated
-        credentials.
-      </p>
+      <ul>
+        <li>
+<a>Verification method</a> rotation is a proactive security measure.
+        </li>
+        <li>
+It is considered a best practice to perform <a>verification method</a> rotation
+on a regular basis.
+        </li>
+        <li>
+Higher security environments tend to employ more frequent verification method
+rotation.
+        </li>
+        <li>
+<a>Verification method</a> rotation applies only to the current or latest
+version of a <a>DID document</a>.
+        </li>
+        <li>
+When a <a>verification method</a> has been active for a long time, or used for
+many operations, a controller might wish to perform a rotation.
+        </li>
+        <li>
+Frequent rotation of a <a>verification method</a> might be frustrating for
+parties that are forced to continuously renew or refresh associated credentials.
+        </li>
+        <li>
+Proofs or signatures that rely on <a>verification methods</a> that are not
```

Comment:
This feels at odds with verbiage in the revised revocation section, which says that omitting a verification method from a DID doc is the only universally supported method of revoking something. How do we square the two statements?

Reply:
I don't understand how it's at odds. It feels aligned. Can you provide some concrete specification text that we could change this to?

Any suggestions @OR13, since it's your language, IIRC?

Reply:
What I'm trying to point out here is that "verification methods that are not present in the latest version of a DID document" are required, in the revocation section, to be interpreted as revoked methods (if they ever existed at all). Thus, this sentence could be reworded to "Verification methods that are already revoked cannot be rotated."